khaos/khaos.go

```diff
@@ -0,0 +1,64 @@
+package khaos
+
+import (
+  "fmt"
+  "math/rand"
+  "time"
+
+  "github.com/30x/khaos-monkey/utils"
+
+  "k8s.io/client-go/1.4/kubernetes"
+  "k8s.io/client-go/1.4/pkg/api"
+)
+
+const (
+  killPodsEventStr = "kill-pods"
+  killServicesEventStr = "kill-services"
+  drainNodeEventStr = "drain-node"
+  targetDaemonsetEventStr = "target-daemonset"
+)
+
+// RunRandomKhaoticEvent randomly picks from the acceptable chaotic events and executes it
+func RunRandomKhaoticEvent(clientset *kubernetes.Clientset, khaosConfig *utils.Config) (err error) {
+  rand.Seed(time.Now().UTC().UnixNano())
+  eventNdx := rand.Intn(len(khaosConfig.KhaoticEvents))
+
+  switch khaosConfig.KhaoticEvents[eventNdx] {
+  case killPodsEventStr:
+    return KillRandomPod(clientset, khaosConfig)
+  case killServicesEventStr:
```

Comment:
Since services aren't managed by a controller do we want to be killing them? I don't see how this would be a recoverable thing.